RELEASE_NOTES.md

```diff
@@ -1,5 +1,54 @@
 # Release Notes
 
+## stable-1.16-2.5.0, stable-1.17-2.5.0, stable-1.18-2.5.0
+
+* Cert-manager
+  - use `delete` upgrade-strategy.
+* Elasticsearch
+  - Fixes regression from [helm/charts&#35;17643](https://github.com/helm/charts/pull/17643) where the explicit selectors do match the previously implicit selectors.
+  - Fix plugin install initcontainer which would fail if plugin already exists. This happens when Node reboots and keeps emptyDir, or if elasticsearch image already contains plugin
+  - In private environments where we replicate all the images, the test image cannot be pulled due it misses imagePullSecrets ([#497](https://github.com/mesosphere/kubernetes-base-addons/pull/497), [@d2iq-dispatch](https://github.com/d2iq-dispatch))
+* Fluent-bit:
+  - bump the fluent-bit app version to 1.5.6
+    - aws: utils: fix mem leak in flb_imds_request
+    - fix double free when destroying connections if the endpoint in unavailable
+    - remove noisy error introduced in v1.5.5
+    - fix deletion of pending connections in the destroy_queue ([#538](https://github.com/mesosphere/kubernetes-base-addons/pull/538), [@d2iq-dispatch](https://github.com/d2iq-dispatch))
+  - changes the update strategy to `delete`. ([#574](https://github.com/mesosphere/kubernetes-base-addons/pull/574), [@dkoshkin](https://github.com/dkoshkin))
+  - Upgrades fluent-bit to v1.5.7. See https://fluentbit.io/announcements/v1.5.7.
```

Comment:
Should probably drop or combine the 1.5.6 notes into 1.5.7

Reply:
I don't know... putting on my sysadmin hat, that wouldn't bother me in the least.